lib/mysql2.rb

```diff
@@ -20,9 +20,12 @@
   end
 
   if dll_path
-    require 'Win32API'
-    LoadLibrary = Win32API.new('Kernel32', 'LoadLibrary', ['P'], 'I')
-    if LoadLibrary.call(dll_path).zero?
+    require 'fiddle'
+    kernel32 = Fiddle.dlopen 'kernel32'
```

Comment:
I think this should still be initial cap `Kernel32` ?

Reply:
Umm, not reusing it are we?

Reply:
I mean the library:`Win32API.new('Kernel32', ...`
vs. `Fiddle.dlopen 'kernel32'`

Reply:
Sorry. between multi-tasking & bouncing between languages, I often miss the obvious.

Actually, the file is `kernel32.dll`, and as you know, Windows file system is case-insensitive (which can cause issues when comparing paths, use `casecmp`...)

Also, ruby/reline essentially created its own `Win32API` class to avoid the deprecation warnings, and it's all lower case.  See [lib/reline/windows.rb
](https://github.com/ruby/ruby/blob/9b18cb341f08c725d74f667ef3724c4cc852fdbd/lib/reline/windows.rb#L55-L62)

Long story short, I think we're ok...